containers/azure-terraform/.devcontainer/devcontainer.json

```diff
@@ -13,17 +13,33 @@
 	},
 	"mounts": [ "source=/var/run/docker.sock,target=/var/run/docker-host.sock,type=bind" ],
 	"overrideCommand": false,
-
+	"runArgs": ["--env-file",".devcontainer/devcontainer.env"],
+    "remoteEnv": {
+		// Sets environment variables required for terrafom remote backend
+		"TF_BACKEND_RESOURCE_GROUP" : "${containerEnv:TF_BACKEND_RESOURCE_GROUP}",
+		"TF_BACKEND_LOCATION": "${containerEnv:TF_BACKEND_LOCATION}",
+		"TF_BACKEND_STORAGE_ACCOUNT" : "${containerEnv:TF_BACKEND_STORAGE_ACCOUNT}",                
+		"TF_BACKEND_CONTAINER" : "${containerEnv:TF_BACKEND_CONTAINER}",
+		"TF_BACKEND_KEY" : "${containerEnv:TF_BACKEND_KEY}",
+		
+		// Sets input variables for terraform templates
+		"TF_VAR_rg_name" : "${containerEnv:TF_VAR_RG_NAME}"
```

Comment:
We should remove ant var related to the sample.

Reply:
Since Terraform recognizes the `TF_VAR` prefix, the intent here is to demonstrate how input can be handled through configuration so that the user does not have to manually input during plan, apply, etc. 

We have commented out the test sample specific variables and added clarification on how input variables can be set for terraform templates.

Reply:
@cmendible - Please let us know if this will work or if we should add any other changes. Thank you so much again for the feedback!

Reply:
LGTM now!